### PR TITLE
Update documentation to add points on PVC binding

### PR DESCRIPTION
**Pull request by vtereso**
# Changes

Update documentation to add points on PVC binding
/cc @spomorski 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: docs/pipelineruns.md [hunk 1]
```diff
@@ -246,6 +246,12 @@ spec:
         claimName: my-volume-claim
 ```
 
+## PersistentVolumeClaims
+
+Any persistent volume claims within a `PipelineRun` will be bound until the
```
Comment (spomorski) [inline docs/pipelineruns.md]: "are bound"

File: docs/pipelineruns.md [hunk 2]
```diff
@@ -246,6 +246,12 @@ spec:
...
+corresponding `PipelineRun` or pods are deleted. The same holds true for
```
Comment (spomorski) [inline docs/pipelineruns.md]: "This also applies to" is a more direct way of conveying this.

**Comment by vtereso** [comment]
@sbwsg @spomorski Can I get an approval?

**Comment by sbwsg** [comment]
thank you!  :pray:

**Comment by bobcatfish** [comment]
Thanks for adding docs @vtereso ! Might make sense to add a link at the top of the doc also so ppl can find the section more easily: https://github.com/tektoncd/pipeline/blob/master/docs/pipelineruns.md#pipelineruns

**Pull request merged**